node/expressjs/middlewares-in-express-js.md

```diff
@@ -0,0 +1,61 @@
+## Middlewares in Express JS
+
+It's been a long time that NODE JS has become a part of JS Ecosystem. Any JS developer Front End or Full Stack would be familiar with Node JS. Any one who has worked on Node would have also worked with Express JS.
+
+So, Express JS is nothing but a framework to help organize your web application into an MVC architecture on the server side. It basically helps you manage everything from routes to handling requests and views for your web application.
```

Comment:
I don't think express.js follows MVC architecture. Can you fact check this. If their docs says MVC anywhere.

Reply:
It says something like MVC Style Controllers. 
https://expressjs.com/en/starter/faq.html.

Reply:
so it's saying that express doesn't have preference of its own. You can choose whatever architecture you want and they have given examples of three different architectures inlcuding MVC